src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -65,8 +65,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     Also, enforce that RuntimeIdentifier is always specified for .NETFramework executables.
   -->
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.RuntimeIdentifierInference.targets" />
-  
-  
+
+  <!-- Import targets from RazorSDK if referenced -->
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.Razor.BeforeCommon.targets" Condition="'$(UsingMicrosoftNETSdkRazor)' == 'true'" />
```

Comment:
Could you put this file under the Microsoft.NET.Sdk.Razor folder?  It should be fine to use a relative import path for it.

Reply:
Done.